tasks/src/reinforcement-learning/about.md

```diff
@@ -23,13 +27,15 @@ You can contribute variants of this task [here](https://github.com/huggingface/h
 **Environment:** The part of the world the agent interacts, comprising everything outside the agent.
 
 
-**State:** Information the agent receives from the environment. In the case of a video game it can be a frame (a screenshot), in the case of a chess playing agent it can be the board position, in the case of a trading agent it can be the price of a certain stock.
+**State:** It is the summary of information presented as the function of history of actions,observations and rewards . In the case of a video game it can be a frame (a screenshot), in the case of a chess playing agent it can be the board position, in the case of a trading agent it can be the price of a certain stock.
 
 
 **Action:** The decision taken by the agent.
 
 
-**Reward:** The numerical feedback signal that the agent receives from the environment based on the chosen action
+**Reward:** The numerical feedback signal that the agent receives from the environment based on the chosen action.
+
+**Observation:** The agent at each step receives a reward and an observation from the environment. This is because the agent can't understand the state of the environment, hence it receives an observation as a result of it's action taken by it at the preceding step.

```

Comment:
There's not many differences between a state and observation. I'd rather not have this.

Reply:
Observation is the information provided by the environment at each time step, whereas stated before a state is actually the function of Observation, Reward and Action taken at each time step.

Reply:
I do understand that for a simplified overview of reinforcement learning the current definitions are insightful, so if that's what the page is going for I don't think the changes I suggested are necessary.